vllm/model_executor/layers/fused_moe/shared_fused_moe.py

```diff
@@ -38,7 +38,6 @@ def __init__(
                 # TODO(wentao): find the root cause and remove this condition
                 self.enable_eplb
                 or (self.moe_config.use_flashinfer_cutlass_kernels and self.dp_size > 1)
-                or self.use_marlin_kernels
```

Comment:
Are we sure this is well tested?
Could you:
1. test with a smaller batch size and get the profile. `vllm bench serve --model deepseek-ai/DeepSeek-V3.1  --dataset-name random --host 127.0.0.1 --port 9256 --random-input-len 2 --random-output-len 10 --request-rate inf --num-prompts 256 --max-concurrency 32`, making sure this works with shared experts.
2. test with accuracy, `lm_eval --model local-completions --model_args "base_url=http://127.0.0.1:9256/v1/completions,model=deepseek-ai/DeepSeek-V3.1,num_concurrent=1024" --tasks gsm8k`

Reply:
Make sense, thanks!